{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/__init__.py

```diff
@@ -1 +1 @@
-
+default_app_config = 'home.apps.MainAppConfig'
```

Comment:
I'm fairly certain we need to do this as `{{ cookiecutter.repo_name }}.$app_name.apps.$config_name`